core/src/main/scala/org/apache/spark/deploy/master/ui/MasterPage.scala

```diff
@@ -375,7 +375,9 @@ private[ui] class MasterPage(parent: MasterWebUI) extends WebUIPage("") {
       <td>{formatResourcesAddresses(driver.resources)}</td>
       <td>{driver.desc.command.arguments(2)}</td>
       {if (showDuration) {
-        <td>{UIUtils.formatDuration(System.currentTimeMillis() - driver.startTime)}</td>
+        <td sorttable_customkey={(-driver.startTime).toString}>
+          {UIUtils.formatDuration(System.currentTimeMillis() - driver.startTime)}
+        </td>
```

Comment:
Why we don't use duration as key? Now it uses negative driver start time as key.

Reply:
For example, for apps, we use duration directly.
```html
<td sorttable_customkey={app.duration.toString}>
  {UIUtils.formatDuration(app.duration)}
</td>
```

Reply:
Driver duration is different from `App` duration because we have `Driver-only Spark job`.

Reply:
Oh, I see. Because for all duration, end time is the same (current time).

Reply:
A single-pod Spark job is the `Driver-only` Spark job without application.

Reply:
Yes, it's because this is `Running Drivers` table.
> Oh, I see. Because for all duration, end time is the same (current time).